algebird-core/src/main/scala/com/twitter/algebird/SketchMap.scala

```diff
@@ -209,6 +209,14 @@ object SketchMap {
 
   def aggregator[K, V](params: SketchMapParams[K])(implicit valueOrdering: Ordering[V], monoid: Monoid[V]): SketchMapAggregator[K, V] =
     SketchMapAggregator(params, SketchMap.monoid(params))
+
+  // TODO: SketchMap's heavy hitters are not strictly associative
+  // (approximately they are)
+  implicit def equiv[K, V]: Equiv[SketchMap[K, V]] =
```

Comment:
I moved these from the tests - are you okay with this, or do you want these to stay in implicit scope in the tests?